pkgs/applications/misc/mkgmap/default.nix

```diff
@@ -9,34 +9,27 @@
 , withExamples ? false
 }:
 let
-  version = "4565";
-  sha256 = "0cfh0msky5812l28mavy6p3k2zgyxb698xk79mvla9l45zcicnvw";
-
   deps = import ./deps.nix { inherit fetchurl; };
   testInputs = import ./testinputs.nix { inherit fetchurl; };
 in
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "mkgmap";
-  inherit version;
+  version = "4600";
 
-  src = fetchsvn {
-    inherit sha256;
-    url = "https://svn.mkgmap.org.uk/mkgmap/mkgmap/trunk";
-    rev = version;
+  src = fetchurl {
```

Comment:
I think we should use fetchzip here.

Reply:
> The main difference between **fetchurl** and **fetchzip** is in how they store the contents. fetchurl will store the unaltered contents of the URL within the Nix store. fetchzip on the other hand will decompress the archive for you, making files and directories directly accessible in the future.

@SuperSandro2000 why? what is the main reason to choose fetchzip over fetchurl in general?

Reply:
with fetchzip you do not need to manually uncrompress the files.

Reply:
With fetchurl you do not need to manually uncompress as well.
What I see is:

> ... This is primarily useful for dynamically generated archives, such as GitHub's /archive URLs, where **the unpacked  content of the zip file doesn't change, but the zip file itself may** (e.g. due to minor changes in the compression algorithm, or changes in timestamps).

So fetchzip is kind of workaround for some cases, where fetchurl doesn't work properly due to mentioned above reasons.